cypress/integration/student_demonstrations_growth.js

```diff
@@ -37,24 +37,25 @@ describe.skip('Student competency growth calculation test', () => {
                 };
 
                 cy.visit('/cbl/dashboards/demonstrations/student').then(() => {
-                    cy.withExt().then(({extQuerySelector}) => {
+                    cy.withExt().then((res) => {
                         const studentUsernames = Object.keys(growthCalculationsByStudent);
                         studentUsernames.forEach(studentUsername => {
                             const studentContentAreas = Object.keys(growthCalculationsByStudent[studentUsername]);
                             studentContentAreas.forEach(studentContentArea => {
                                 cy.visit(`/cbl/dashboards/demonstrations/student#${studentUsername}/${studentContentArea}`);
+                                cy.request(`/cbl/dashboards/demonstrations/student#${studentUsername}/${studentContentArea}`).then((res) => {
+                                  expect(res.status).to.eq(200)
+                                })
                                 // ensure that API has loaded required data
                                 cy.wait('@studentCompetencyData')
-                                    .its('status')
-                                    .should('be', 200)
```

Comment:
I suggest we revert this part back to the original code, with one minor change: 
*Line 49*
Replace: `.should('be', 200)`
With: `.should('eq', 200)`

Reply:
This code was doing something different than what maybe the `cy.request` you added was aiming to do. `cy.wait` waits for the application loaded into the page to make an API call on its own, and lets you check the status of that. This accomplishes two things for us here: has the test check that an API call we expect the page to make succeeded, but also serves to make the test script wait for the API call to finish before the next assertion (otherwise the next assertion might time out if the API call is slow some time)

Reply:
Got it